JSDemos/Demos/Scheduler/Timelines/description.md

```diff
@@ -1 +1,10 @@
-This demo illustrates the collection of&nbsp;timeline views available in&nbsp;the Scheduler component. Appointments are associated with several resources (an&nbsp;appointment color depends on&nbsp;the associated resource) and grouped by&nbsp;one of&nbsp;them.
\ No newline at end of file
+Timeline views display appointments as a sequence of events on a horizontal timeline, as opposed to basic views that display appointments in a calendar-like fashion. You can use the View Switcher above the timetable to switch between the views.
+
+To add timeline views to your Scheduler, list them in the [views](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array. You can also set the initially selected view in the [currentView](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#currentView) property.
+
+For information about all supported view types, refer to the [View Types](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/) help topic.
+
+In this demo, appointments are colored and grouped according to their resources. For information on how to configure resources and group appointments by them, view the following demos:
```

Comment:
What we call "grouping by resources", usual people will likely consider "side-by-side calendars" or "multiple calendars". Outlook never really calls them "resources". See the following article, as an example:

https://support.microsoft.com/en-us/office/view-multiple-calendars-at-the-same-time-fffa8783-0556-4ea1-ba62-3ed8a95a903c

So when we introduce this concepts, we should start with a simple phrase and only then switch to technical terminology. 

```suggestion
This demo shows multiple calendars side-by-side (groups data by resources). For information on how to configure such a display, see the following demos:
```